INSTALL/Install_configure_postgreSQL_CENTOS7.md

```diff
@@ -88,8 +108,8 @@ for some reason if you wish to delete the user. do the following **as postgres u
 ```
 dropuser haas
 ```
-@@Note@@: Make sure that the database role you create corresponds to an existing system user. 
-eg. There has to be a system user `haas` to access database role `haas`.
+**Note**: Make sure that the database role you create corresponds to an existing system user. 
```

Comment:
same objection. for this configuration we had a database role that corresponds to a system role, but as long as the database ownership and information in the configuration file are correct this is not necessary (at least with my experience using a development environment)